.github/workflows/molecule.yaml

```diff
@@ -27,6 +27,7 @@ jobs:
         - edpm_logrotate_crond
         - edpm_multipathd
         - edpm_network_config
+        - edpm_neutron_dhcp
```

Comment:
Thanks for enabling the molecule.

As discussed elsewhere, I think we should add a check in molecule scenario that would validate sidecar are not cleaned up when agent(s) are down.

For this, you would 

- make sure the agent container is up (already done in test)
- create qdhcp- (or metadata equivalent) namespace from inside the agent container
- start dnsmasq (or metadata equivalent) via wrapper (calling /usr/local/bin/<service> from the agent container)
- stop the agent container
- check that qdhcp- (or metadata equivalent) namespace is up and the service process is up